step.yml

```diff
@@ -199,8 +199,8 @@ inputs:
       category: Debug
       title: Output tool
       description: |-
-        If output_tool is set to xcpretty, the xcodebuild output will be prettified by xcpretty.
-        If output_tool is set to xcodebuild, the raw xcodebuild output will be printed.
+        If set to `xcpretty`, the xcodebuild output will be prettified by xcpretty.
+        If set to `xcodebuild`, the raw xcodebuild output will not be visible in the build log.
```

Comment:
I would prefer mentioning the last 20 lines will be printed and the completel log will be available only if they have deploy-to-bitrise-io step added.